x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.tsx

```diff
@@ -88,38 +95,50 @@ export class TiledVectorLayer extends VectorLayer {
     await this._syncMVTUrlTemplate(syncContext);
   }
 
-  _syncSourceBindingWithMb(mbMap: unknown) {
-    // @ts-ignore
-    const mbSource = mbMap.getSource(this.getId());
-    if (!mbSource) {
-      const sourceDataRequest = this.getSourceDataRequest();
-      if (!sourceDataRequest) {
-        // this is possible if the layer was invisible at startup.
-        // the actions will not perform any data=syncing as an optimization when a layer is invisible
-        // when turning the layer back into visible, it's possible the url has not been resovled yet.
-        return;
-      }
+  // getId() {
+  //   return this._descriptor.id + DELIMITER + this._source.getLayerName() + DELIMITER;
+  // }
 
-      const sourceMeta: MVTSingleLayerVectorSourceConfig | null = sourceDataRequest.getData() as MVTSingleLayerVectorSourceConfig;
-      if (!sourceMeta) {
-        return;
-      }
+  _getMbSourceId(): string {
```

Comment:
Should this be moved up into abstract layer implementation? There are lots of places where other layers use `getId` as mbSourceId.